src/components/id3_tag.rs

```diff
@@ -249,6 +250,25 @@ impl AudioTagEdit for Id3v2Tag {
     fn remove_genre(&mut self) {
         self.inner.remove_genre();
     }
+
+    fn comment(&self) -> Option<&str> {
+        for comment in self.inner.comments() {
+            if comment.description.is_empty() {
+                return Some(comment.text.as_str());
+            }
+        }
+        None
+    }
+    fn set_comment(&mut self, comment: String) {
+        self.inner.add_frame(id3::frame::Comment {
+            lang: "".to_string(),
```

Comment:
```suggestion
            lang: "XXX".to_string(),
```
According to https://mutagen-specs.readthedocs.io/en/latest/id3/id3v2.4.0-structure.html:
> The three byte language field, present in several frames, is used to describe the language of the frame’s content, according to ISO-639-2 [ISO-639-2]. The language should be represented in lower case. If the language is not known the string “XXX” should be used